R/utils.R

```diff
@@ -182,3 +182,62 @@ is_installed <- function(path_pkg = ".") {
 is_windows <- function() {
   identical(.Platform$OS.type, "windows")
 }
+
+#' Add library directory
+#'
+#' @description Add directories that need to be available when running
+#'   `script.R`. During [benchmark_run_ref] they will be placed in the
+#'    same directory as `script.R`.
+#' @param ... A number of directories, as strings in relation to the current
+#'   working directory, that contain scripts you want to source in `script.R`.
+#' @return The temp dir invisibly.
+#' @examples
+#' \dontrun{
+#' # In script.R
+#' add_lib_dirs(c("bench", "inst/scripts"))
+#'
+#' source("scripts/setup.R")
+#'
+#' touchstone::benchmark_run_ref(
+#'   expr_before_benchmark = {
+#'     !!setup
+#'     source("bench/exprs.R")
+#'   },
+#'   run_me = some_exprs(),
+#'   n = 6
+#' )
+#' }
+#' @export
+add_lib_dirs <- function(...) {
+  temp_dir <- getOption("touchstone.temp_dir")
+
+  if (is.null(temp_dir)) {
+    usethis::ui_stop(c(
+      "Temporary directory not found. ",
+      "This function is only for use within 'script.R'."
```

Comment:
I think important is to say that `script.R` must be called with `run_script()`. Can you add that?